src/_typography.css

```diff
@@ -120,22 +120,21 @@ p {
   font-kerning: none;
 }
 
-.letter-rule:first-letter {
-  border-top: var(--space-0) solid var(--red);
-  padding-top: var(--space-2);
-}
-
+.letter-rule:before,
```

Comment:
Because you are using `:before` instead of `:first-letter`, the height of this element will contribute to the height of the parent element, which it did not before. As a result, you will need to adjust the existing pages that use `.letter-rule` (solutions pages) to make sure the current vertical spacing is preserved. Not something to include in this PR, just be aware when you bump the version on envoy.com

Reply:
Thanks for the info!